### ISSUE TITLE
Show the right globe depending on user's language

### ISSUE THREAD

**Issue by tleydxdy** [vector-im/element-android#3380]
#### Is your feature request related to a problem? Please describe.
With the new Element android update, I find that public rooms is marked by a globe that shows America. I realized this was also a problem on the web version as well.

#### Describe the solution you'd like.
It would be really cool if this would change to match the user's language/location.

#### Describe alternatives you've considered.
Arguably the 🌐 icon would fit better also, since this is on the internet after all.

**Comment by bmarty** [comment]
I agree. I think we should fix the issue by using a totally different picto. A earth globe is not given any public/private hint (to me at least).
The design team will have to think about that, but they have many other priorities.
